### PR TITLE
vendor-k8s-1-13

### PR DESCRIPTION
**Pull request by calvix**
vendor k8s 1.13 into aws operator

towards https://github.com/giantswarm/giantswarm/issues/4875

### CODE REVIEW

File: service/controller/v23/version_bundle.go [hunk 1]
```diff
@@ -8,15 +8,25 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.13.3",
```
Comment (teemow) [inline service/controller/v23/version_bundle.go]:
```suggestion
				Description: "Update kubernetes to 1.13.3. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md",
```

**Comment by calvix** [comment]
RFR

**Comment by teemow** [comment]
@calvix shall we roll in Container Linux 1967.6.0

https://coreos.com/releases/#1967.6.0

Not sure if this really affects us but this was fixed:
https://github.com/coreos/bugs/issues/2549

**Comment by calvix** [comment]
@teemow I have not seen the issue so it might not affect us, anyway, this is about k8s 1.13, we can address CoreOS in other PR if necessary

**Pull request merged**